medusa/server/api/v2/config.py

```diff
@@ -430,3 +507,16 @@ def data_statuses():
         section_data['strings'] = common.statusStrings
 
         return section_data
+
+    @staticmethod
+    def data_metadata():
+        """Metadata."""
+        section_data = NonEmptyDict()
+
+        section_data['metadataProviders'] = NonEmptyDict()
```

Comment:
@p0psicles 
Is there any reason for it being nested?
The metadata can just be the providers, right?

Reply:
To leave room for other gneral metadata config. For example applying to all providers